spec/domain/person/filter/role_spec.rb

```diff
@@ -347,6 +348,20 @@ def filter(attrs)
           role.update(created_at: now)
           expect(filter(start_at: now, finish_at: now).entries).to have(1).item
         end
+
+        context 'excluding archived' do
+          it 'does not find role archived within range' do
+            role.update_attribute(:archived_at, 1.day.ago)
+            expect(filter(start_at: now, finish_at: now, include_archived: false).entries).to be_empty
+          end
+        end
+
+        context 'including archived' do
+          it 'finds role archived within range' do
```

Comment:
Basierend auf kürzlich aufgetretenen [Problemen](https://github.com/hitobito/hitobito/pull/1897) mit OR-Bedingungen in SQL Queries, könnte es sich lohnen hier einen Test `it 'does not find archived role outside range'` zu schreiben, um sicherzugehen dass die Archivierungslogik nicht die sonstigen Query-Bedingungen aushebelt?